lib/decidim/budgets/component.rb

```diff
@@ -0,0 +1,202 @@
+  end
+
+  component.register_stat :projects_count, priority: Decidim::StatsRegistry::LOW_PRIORITY do |components, start_at, end_at|
+    Decidim::Budgets::FilteredProjects.for(components, start_at, end_at).count
+  end
+
+  component.register_stat :orders_count do |components, start_at, end_at|
+    budgets = Decidim::Budgets::Budget.where(component: components)
+    orders = Decidim::Budgets::Order.where(budget: budgets)
+    orders = orders.where("created_at >= ?", start_at) if start_at.present?
+    orders = orders.where("created_at <= ?", end_at) if end_at.present?
+    orders.count
+  end
+
+  component.register_stat :comments_count, tag: :comments do |components, start_at, end_at|
+    projects = Decidim::Budgets::FilteredProjects.for(components, start_at, end_at)
+    projects.sum(:comments_count)
+  end
+
+  component.register_stat :followers_count, tag: :followers, priority: Decidim::StatsRegistry::LOW_PRIORITY do |components, start_at, end_at|
+    projects_ids = Decidim::Budgets::FilteredProjects.for(components, start_at, end_at).pluck(:id)
+    Decidim::Follow.where(decidim_followable_type: "Decidim::Budgets::Project", decidim_followable_id: projects_ids).count
+  end
+
+  component.exports :projects do |exports|
+    exports.collection do |component_instance, _user, resource_id|
+      budgets = resource_id ? Decidim::Budgets::Budget.find(resource_id) : Decidim::Budgets::Budget.where(decidim_component_id: component_instance)
+      Decidim::Budgets::Project
+        .where(decidim_budgets_budget_id: budgets)
+        .includes(:category, :component)
+    end
+
+    exports.include_in_open_data = true
+
+    exports.serializer Decidim::Budgets::ProjectSerializer
+  end
+
+  component.settings(:global) do |settings|
+    settings.attribute :scopes_enabled, type: :boolean, default: false
+    settings.attribute :scope_id, type: :scope
+    settings.attribute :workflow, type: :enum, default: "one", choices: -> { Decidim::Budgets.workflows.keys.map(&:to_s) }
+    settings.attribute :projects_per_page, type: :integer, default: 12
+    settings.attribute :vote_rule_threshold_percent_enabled, type: :boolean, default: true
+    settings.attribute :vote_threshold_percent, type: :integer, default: 70
+    settings.attribute :vote_threshold_percent, type: :integer, default: 70
+    settings.attribute :vote_rule_minimum_budget_projects_enabled, type: :boolean, default: false
+    settings.attribute :vote_minimum_budget_projects_number, type: :integer, default: 1
+    settings.attribute :vote_rule_selected_projects_enabled, type: :boolean, default: false
+    settings.attribute :vote_selected_projects_minimum, type: :integer, default: 0
+    settings.attribute :vote_selected_projects_maximum, type: :integer, default: 1
+    settings.attribute :comments_enabled, type: :boolean, default: true
+    settings.attribute :comments_max_length, type: :integer, required: false
+    settings.attribute :geocoding_enabled, type: :boolean, default: false
```

Comment:
I don't see the checkbox to enable the geocoding in the budget component creation or edit form